custom/ewsghana/static/ewsghana/ko/ews_filter.drilldown_options.js

```diff
@@ -0,0 +1,150 @@
+//probably we need clear this file
+//copy/paste from https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/reports/static/reports/ko/report_filter.drilldown_options.js
```

Comment:
This comment seems to not refer to the whole file. For example I can't find the `ko.bindingHandlers.select2 = ...` section in the linked page. Should this comment be moved or removed?